README.rst

```diff
@@ -918,6 +918,32 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+Byte Adaptive Measuring in Charts
+---------------------------------
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/chartsize.png
+   :align: center
+
+This behavior helps us to show the data and unit of the chart in a more readable way,
+the units are shown in `B`, `KB`, `MB` and `GB` respectively, and datapoints have
+same unit to maintain consistency in the chart, summarycircles have different units
+and are shown in `B`, `KB`, `MB` and `GB` to overview chart summary in a more 
```

Comment:
if you can implement the TB as I am asking, here and below we'll have to update it to mention it.